modules/reference/pages/users/user-details.adoc

```diff
@@ -48,6 +48,10 @@ To delete a user:
 
 For instructions to deactivate your own account, see xref:reference:home/user-account-account-deactivation.adoc[].
 
+[NOTE]
+====
+If you delete a user account, any schedules that user created will also be deleted.
```

Comment:
I would avoid the term "schedules", as it is not clear which schedules we mean here. I'd suggest:
- either using "recurring actions" term,
- or referencing the "recurring-actions" section in the ref guide.
- (alternatively: both :smile:  )